src/test/java/com/revature/services/impl/BatchServiceImplTest.java

```diff
@@ -24,15 +25,15 @@
 	
 	@Mock
 	private BatchRepository br;
-
+	
```

Comment:
The changes in this file seem irrelevant. Please remove.

Reply:
Reverted irrelevant changes.